src/plugins/dashboard/public/embeddable/dashboard_container_factory.tsx

```diff
@@ -43,21 +43,18 @@ interface StartServices {
   uiActions: UiActionsStart;
 }
 
-export class DashboardContainerFactory extends EmbeddableFactory<
-  DashboardContainerInput,
-  ContainerOutput
-> {
+export class DashboardContainerFactory
+  implements
+    EmbeddableFactoryDefinition<DashboardContainerInput, ContainerOutput, DashboardContainer> {
   public readonly isContainerType = true;
   public readonly type = DASHBOARD_CONTAINER_TYPE;
 
-  constructor(private readonly getStartServices: () => Promise<StartServices>) {
-    super();
-  }
+  constructor(private readonly getStartServices: () => Promise<StartServices>) {}
 
-  public async isEditable() {
+  public isEditable = async () => {
     const { capabilities } = await this.getStartServices();
     return !!capabilities.createNew && !!capabilities.showWriteControls;
-  }
+  };
 
   public getDisplayName() {
```

Comment:
Looks like this method should also be instance property

```suggestion
  public readonly getDisplayName = () => {
```